src/PoCompiler.php

```diff
@@ -254,17 +254,9 @@ protected function buildProperty($property, $value, $obsolete = false)
      */
     protected function cleanExport($string)
     {
-        $quote   = '"';
-        $slash   = '\\';
-        $newline = "\n";
+        $string = sprintf('"%s"', addcslashes($string, "\42\134")); // " and \
```

Comment:
As explained below, `\42\134` is hard to understand, what and why this exact value?

Reply:
You're totally right. I've fixed that. See latest commit.